packages/aws-cdk-lib/aws-ec2/lib/restrict-default-security-group-handler/index.ts

```diff
@@ -69,8 +69,20 @@ async function onUpdate(event: AWSLambda.CloudFormationCustomResourceUpdateEvent
  * Revoke both ingress and egress rules
  */
 async function revokeRules(groupId: string, account: string): Promise<void> {
-  await ec2.revokeSecurityGroupEgress(egressRuleParams(groupId));
-  await ec2.revokeSecurityGroupIngress(ingressRuleParams(groupId, account));
+  try {
+    await ec2.revokeSecurityGroupEgress(egressRuleParams(groupId));
+  } catch (e: any) {
+    if (!(e instanceof Error) || (e instanceof Error && e.name !== 'InvalidPermission.NotFound')) {
```

Comment:
Thanks for your explanation. I removed error type checks.
Just found the error code `InvalidPermission.NotFound` is listed in this [doc](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/errors-overview.html#api-error-codes-table-client).

> We should just be able to consolidate like so?

I think these two API calls should be put into separate try-catch blocks, because even if the default egress rule is not found, we still want to execute `revokeSecurityGroupIngress`.